lib/change.subscriber.ts

```diff
@@ -23,7 +23,8 @@ export class ChangeSubscriber implements EntitySubscriberInterface<any> {
     if (ChangeSubscriber.disabled)
       return;
 
-    if (Reflect.hasMetadata("__track_changes", event.entity.constructor)) {
+    if (event.entity &&
+				Reflect.hasMetadata("__track_changes", event.entity.constructor)) {
```

Comment:
Is this a linting error or purposely on a new line?

Reply:
It's a new line on purpose - since the second line is kind of long, I think it helps readability.

However, note that the weird, long spacing on that second line is an artifact of how GH renders tabs (this project uses tabs for indentation, not spaces). So, if you open this in VSCode, it actually looks like this:

![Tabs look OK on VSCode](https://user-images.githubusercontent.com/1930146/201452416-e4bb106d-95b2-4ce7-86ba-2a1243d21406.png)

Reply:
Lets keep style based changes out of this PR for now, the next line is even longer, so this change would only make sense if we were to apply this style to the entire project.